pip-dep/requirements.txt

```diff
@@ -10,7 +10,7 @@ Pillow~=6.2.2
 requests~=2.22.0
 RestrictedPython~=5.0
 scipy~=1.4.1
-syft-proto~=0.3.0
+syft-proto~=0.3.1
```

Comment:
We can go for the 0.4.1 which got released after this one

Reply:
Hmm, first I want to merge this [branch](https://github.com/OpenMined/PySyft/pull/3453) and then this should be fixed :D